src/drivers/vmount/vmount.cpp

```diff
@@ -377,7 +377,7 @@ static int vmount_thread_main(int argc, char *argv[])
 			}
 
 			hrt_abstime now = hrt_absolute_time();
-			if (now - last_output_update > 10000) { // rate-limit the update of outputs
+			if (now - last_output_update > 10000 || control_data != nullptr) { // rate-limit the update of outputs
```

Comment:
This is necessary because for inputs that only seldom produce non-null control data (e.g. `InputMavlinkROI` when a new ROI mission item is processed) it was basically a gamble whether that control data got through to the output.

A potential problem is now with inputs that produce a lot of control data (e.g. `InputRC`, which produces them at the rate of `manual_control_setpoint`). Can somebody provide some feedback if this is problematic? What is max rate we expect to see on `manual_control_setpoint`?

Reply:
About 50 Hz.

Reply:
Do you think that's problematic?